msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -142,31 +142,32 @@ In-meeting notification must not use task module. Task module isn't invoked in a
 
 Shared meeting stage allows meeting participants to interact with and collaborate on app content in real time. You can share your apps to the collaborative meeting stage in the following ways:
 
-* [Share entire app to stage](#share-entire-app-to-stage) using the share to stage button in Teams client.
+* [Share entire app to stage](#share-entire-app-to-stage) using the share to stage button in Teams client or through [deeplinks](~/concepts/build-and-test/deep-links.md#generate-a-deep-link-to-share-content-in-meetings).
```

Comment:
can we link to the specific section  'Generate a deep link to share content to stage in meetings'  in deeplinks page?